### PR TITLE
Remove example script which breaks install

### PR DESCRIPTION
**Pull request by kirkins**
According to this discussion https://github.com/VundleVim/Vundle.vim/issues/713 this plugin is simply an example, but the example seems to consistently break the installation process.

If this is the case better to remove it.

Another example of it causing problems: https://github.com/VundleVim/Vundle.vim/issues/784

### CODE REVIEW

**Comment by ryanoasis** [comment]
Yeah... this seems to be causing way too many unnecessary issues :disappointed:

**Comment by ryanoasis** [comment]
I think the purpose was to show varying formats of plugins support. May I suggest commenting out the `L9` lines instead of entirely removing them?

File: README.md [hunk 1]
```diff
@@ -61,8 +61,6 @@
    " Keep Plugin commands between vundle#begin/end.
    " plugin on GitHub repo
    Plugin 'tpope/vim-fugitive'
-   " plugin from http://vim-scripts.org/vim/scripts.html
```
Comment (ryanoasis) [inline README.md]: Instead of removing I think a better way would be to comment out the next line and the `ascenator/L9` line.

**Comment by kirkins** [comment]
That makes sense :+1:

**Comment by ryanoasis** [comment]
If you can make the suggested changes I will merge 😄

**Comment by kirkins** [comment]
right! how does that look?

**Comment by ryanoasis** [comment]
Looks good, just one thing I would comment out the other L9 example too:
Plugin 'ascenator/L9', {'name': 'newL9'}

**Comment by kirkins** [comment]
Got it

**Comment by ryanoasis** [comment]
Thanks. Looks good to me. This should cut down on the L9 related reported issues :smile:

**Pull request merged**